news/pytest-migration.rst

```diff
@@ -0,0 +1,11 @@
+**Added:** xonsh_builtins, xonsh_execer fixtures in conftest.py
```

Comment:
Small formatting issue here for the changelog amalgamater -- things should be bullet pointed with a single `*` so this should read:

```console
**Added:**

* xonsh_builtins, xonsh_execer fixtures in conftest.py
```